### ISSUE TITLE
Remove jQuery

### ISSUE THREAD

**Issue by MrPowerGamerBR** [LorittaBot/Loritta#2106]
We don't really need jQuery anymore, but some parts of the website still uses it.

There are a few dependencies that we need to replace before we can drop jQuery:
* [ ] toastr (Only used in old dashboard pages)
* [ ] autosize (Used to automatically scale textboxes)
* [ ] select2 (Used for custom dropdowns, that's the hardest to drop)

**Comment by MrPowerGamerBR** [comment]
Superseded by https://github.com/LorittaBot/Loritta/issues/2226 because we will rewrite the website anyway
